includes/validation/class-amp-validation-manager.php

```diff
@@ -265,10 +265,14 @@ public static function init() {
 	/**
 	 * Filter AMP options to set Standard template mode if it is an AMP-override request.
 	 *
-	 * @param array $options Options.
-	 * @return array Filtered options.
+	 * @param array|false $options Options.
+	 * @return array|false Filtered options.
```

Comment:
Isn't the return value now always made an array?

```suggestion
	 * @return array Filtered options.
```